### DISCUSSION TITLE
1.1: Has anyone provably completed this whole 50+ hour course in a single weekend?

### DISCUSSION THREAD

**Discussion by tippi-fifestarr** [Cyfrin/foundry-full-course-cu#1196] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/62179036/f2ec879f-f34c-44e0-a6ea-74c077f561ea)
Seems almost impossible but funny to imagine. It's great that you encourage people to use spaced repetition to make gains 💪 

That said, immersion can take academic knowledge into the realm of experiential learning, which is the bees knees.

I want people to reply to this, at the beginning of their journeys.  
Come back again at the end and share with us a screenshot of your first reply and tell the community about your adventure in Updraft!

**Comment by xyz899** [comment]
Okay,
So I'll let you think twice about what you said. You said you find this funny to finish the 50h+ course in a single weekend but tell me how exactly is that possible when you have 48 hours ? Plus, Working with extra focus for 24hours is not possible and not recommended at all for health matters. Programming is a very prosperous profession and industry but search what happens for someone sitting on a chair all day long

**Reply by tippi-fifestarr** [reply]
Hey @Winter9998, thanks for your thoughtful response!  I thought twice and even thrice when I realized your wisdom score is "very nice". First off, you're absolutely right about the practical and health considerations of trying to marathon through a course like this in a weekend. It was more of a tongue-in-cheek comment to spark some imagination and humor in our learning journey. 😄 But if someone wants to watch everything at 1.5x speed while drinking root beer and liquid cheese, I cannot stop them and might just cheer them on.  Who knows, maybe you only have a weekend before starting a new project and just need to load a bunch of blocks into your BIOS 👩‍💻  

I definitely advocate for a balanced and healthy approach to learning, as you pointed out. It's important to take breaks, stay active, and not overstrain ourselves. Your point about the physical challenges of long hours at a desk is super relevant, and a great reminder for all of us to take care of our health as we dive into this course.

I'm curious to hear from others too! What are your strategies for balancing intensive learning with health and wellbeing? And for those just starting out, what are your thoughts as you begin this adventure in Updraft? Let's keep this discussion going and support each other on this educational journey!

**Reply by sorenabirckof** [reply]
My coding life at the company basically eats up everything until lunchtime, so I only have the afternoons and evenings for my blockchain adventure. Mornings are tight, but I manage to squeeze in 4-5 hours of solidity learning in the afternoons, even if it's not all sunshine and roses. I'm not talking marathon sessions with zero breaks – my magic number is 15 minutes of intense focus followed by a quick refresh. A walk, a cuppa, a phone call with a friend or colleague - anything to give my brain a breather. Social media is a big no-no during these breaks, though. Once that rabbit hole opens, my learning time goes down the drain! So yeah, I'm currently diving headfirst into the basics of Solidity, which is totally new territory for me. No prior blockchain knowledge, nada. So for now, it's all about laying the groundwork, mastering the fundamentals, and getting my head around the theory of this whole fascinating universe.

**Comment by tobezhanabi** [comment]
lol. I think if you can get compound V from THE Boys you can do it.
